en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-overview.md

```diff
@@ -0,0 +1,31 @@
+# Salesforce REST Connector Overview
+
+This documentation provides overall mapping with each sections in Salesforce connector and a inbound endpoint documentation. Click the links of the shown bellow to view the documentation for **Salesforce REST Connector** and **Salesforce Inbound Endpoint**.
+
+## Salesforce REST Connector
```

Comment:
Add some information about what exactly can be done using this connector.

Reply:
Added.